queries.md

```diff
@@ -4,29 +4,196 @@
 
 ## 1. Find all the companies that include 'Facebook' on the **name** field.
 
- - **`query`**: {name: 'Facebook'}
+{
+ filter: {
+  name: 'Facebook'
+ }
+}
  
- ## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
+## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
 
- - **`query`**: {category_code: 'web'}
- - **`projection`**: {name: 1, _id: 0}
+{
+ filter: {
+  category_code: 'web'
+ },
+ project: {
+  name: 1,
+  _id: 0
+ }
+}
 
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
 
+{
+ filter: {
+  name: 'Twitter'
+ },
+ project: {
```

Comment:
They never asked for the ID to show... So you are missing ` _id: 0`... Then it would become:
`{name: 1, category_code:1, founded_year:1, _id: 0}`